### PR TITLE
[5287] Code editor doesn't focus on the edit panel

### PR DESCRIPTION
**Pull request by aachavarria**
https://github.com/craftercms/craftercms/issues/5287
https://github.com/craftercms/craftercms/issues/5266

### CODE REVIEW

**Review by rart — COMMENTED** [review]
PRs got mixed. Please remove the LogConsole work from this PR.

File: ui/app/src/components/CodeEditorDialog/CodeEditorDialogContainer.tsx [hunk 1]
```diff
@@ -233,6 +233,7 @@ export function CodeEditorDialogContainer(props: CodeEditorDialogContainerProps)
         <ConditionalLoadingState isLoading={loading} classes={{ root: classes.loadingState }}>
           <AceEditor
             ref={editorRef}
+            autoFocus
```
Comment (rart) [inline ui/app/src/components/CodeEditorDialog/CodeEditorDialogContainer.tsx]: 1. What happens to this flag when opening readonly?
2. When you go from ready only to edit, does the autoFocus pick up?

**Review by rart — CHANGES_REQUESTED** [review]
Let's try adding a `default` background to the editor when in readonly; make it more distinct from the editable editor

File: ui/app/src/components/CodeEditorDialog/CodeEditorDialogContainer.tsx (hunk 1 above)
Comment (aachavarria) [reply ui/app/src/components/CodeEditorDialog/CodeEditorDialogContainer.tsx]: checking

**Review by rart — APPROVED** [review]

**Pull request merged**